packages/server/postgres/migrations/1725438336254_addOrganizationUserUniqueConstraint.ts

```diff
@@ -0,0 +1,45 @@
+import {Kysely, PostgresDialect, sql} from 'kysely'
+import getPg from '../getPg'
+
+export async function up() {
+  const pg = new Kysely<any>({
+    dialect: new PostgresDialect({
+      pool: getPg()
+    })
+  })
+
+  // get rid of duplicates
+  // no kysely here, because I tested the SQL directly and don't want to touch it
+  await sql`
+    DELETE FROM "OrganizationUser" d USING "OrganizationUser" k
+    WHERE d."userId" = k."userId"
+      AND d."orgId" = k."orgId"
+      AND (
+        -- keep non-removed over removed
+        (k."removedAt" IS NULL AND d."removedAt" IS NOT NULL)
+        -- or removed later
+        OR (k."removedAt" IS NOT NULL
+           AND d."removedAt" IS NOT NULL
+           AND (k."removedAt" > d."removedAt"
+               OR ((k."removedAt" = d."removedAt") AND k.id > d.id)
+           )
+        )
+        -- or newer non-removed
+        OR (k."removedAt" IS NULL AND d."removedAt" IS NULL AND k.id > d.id)
+      );
+  `.execute(pg)
+
+  await pg.schema
+    .alterTable('OrganizationUser')
+    .addUniqueConstraint('unique_org_user', ['orgId', 'userId'])
```

Comment:
back in the days of proration, we needed to keep track of `removedAt` since some users would join an org, then leave it, then join again & we didn't want to treat them as a new user with a grace period.

I see the unique constraint doesn't include `removedAt`, which means only 1 row per user per org, so if a user joins, then leaves, then joins again, what happens?
The logic here would say `removedAt` doesn't get reset: https://github.com/ParabolInc/parabol/blob/63bf930989f1493f54d36407a990863519b80528/packages/server/billing/helpers/adjustUserCount.ts#L86. At least I think, I didn't actually test this yet